src/mainwindow.cpp

```diff
@@ -51,6 +59,19 @@ void MainWindow::setupWebView() {
                 });
         notification->sendEvent();
       });
+#else
+QWebEngineProfile::defaultProfile()->setNotificationPresenter(
+    [&](std::unique_ptr<QWebEngineNotification> notificationInfo) {
+      QByteArray title_ba = notificationInfo->title().toLocal8Bit();
+      QByteArray message_ba = notificationInfo->message().toLocal8Bit();
+      char image_path[1024], command[2048];
+      const char *title = title_ba.data();
+      const char *message = secureMe(message_ba.data());
+      snprintf(image_path, 32+strlen(title), "/tmp/discord-screenaudio-%s.png", title);
+      notificationInfo->icon().save(image_path);
+      snprintf(command, 70+strlen(image_path)+strlen(title)+strlen(message), "notify-send --icon=\"%s\" --app-name=\"discord-screenaudio\" \"%s\" \"%s\"", image_path, title, message);
+      system(command);
+    });
 #endif
```

Comment:
You could move the `#ifdef KNOTIFICATIONS` into the lambda to make it a bit less redundant.

Reply:
If you know how, do it. I don't know what lambda is, to be honest.

Reply:
I am just talking about the callback function passed to `setNotificationPresenter`. I've pushed the required changes.